.github/workflows/main.yml

```diff
@@ -0,0 +1,48 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: true
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '2.7'
```

Comment:
Perhaps it would be good to upgrade to Ruby 3 (since Ruby 2 is EOL according to https://endoflife.date/ruby and https://www.ruby-lang.org/en/downloads/branches/), but the purpose of this PR is just to migrate from Travis without much additional changes and the Travis pipeline [was running on Ruby 2.7](https://app.travis-ci.com/github/kaitai-io/kaitai_struct_doc/jobs/598856275#L269):

```console
$ ruby --version
ruby 2.7.6p219 (2022-04-12 revision c9c2245c0a) [x86_64-linux]
```

After we see GitHub Actions working, we can look into upgrading to Ruby 3.

Reply:
Yep, makes sense, let's do that as a separate update.